### PR TITLE
Homework week2

### PR DESCRIPTION
**Pull request by eeeibrahimsahin**

### CODE REVIEW

File: Week2/homework/js-exercises/1_removeComma.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+let myString = 'hello,this,is,a,difficult,to,read,sentence';
+console.log(myString.length);
+//first way: I learned this way from stackoverflow. it benefits to learn 'regular expression' for me.
+myString = myString.replace(/,/g,' '); //regular expression with global flag
+console.log(myString);
+//second way is my way:)
+myString= myString.split(',')[0];
+console.log(myString);
```
Comment (wilgert) [inline Week2/homework/js-exercises/1_removeComma.js]: Good that you came up with your own solution! Although it looks like your way works, it does not really. Try removing line 5 and see what happens. Then try to think how you can prevent a bug like this in the future.

File: Week2/homework/js-exercises/2_oddEvenReporter.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+for(let i = 0; i <= 20; i++){
+    if(i%2==0){
+        console.log(`The numar ${i} is even!`);
```
Comment (wilgert) [inline Week2/homework/js-exercises/2_oddEvenReporter.js]: Very nice! Can you make it so the string  `The number ${i} is` is not duplicated?

File: Week2/homework/js-exercises/3_recipeCard.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            console.log(`${i}:`);
+            for (let item of list[i]){
+                console.log(item)
+            }
+        }else{
+            console.log(`${i}: ${list[i]}`);
+        }
+    }
+    console.log('-----------------------------');
+}
```
Comment (wilgert) [inline Week2/homework/js-exercises/3_recipeCard.js]: Very nice! I've got one more challenge for you, can you make it so it looks even more like what was asked in the exercise:

```
Meal name: Omelete
Serves: 2
Ingredients:
4 eggs
2 strips of bacon
1 tsp salt/pepper
```

Hint: you should not change the names of the properties in your objects, but solve it in another way.

File: Week2/homework/js-exercises/4_readBooks.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+const books = [{bookName:'Where the Crawdads Sing',
```
Comment (wilgert) [inline Week2/homework/js-exercises/4_readBooks.js]: minor formatting issue: `bookName` should be on the next line.

File: Week2/homework/js-exercises/5_whoThirsty.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
+'use strict';
```
Comment (wilgert) [inline Week2/homework/js-exercises/5_whoThirsty.js]: very nice!

File: Week2/homework/js-exercises/6_gradeConvertor.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
+'use strict';
+const yourScore = 74;
+// this command shows the grade and the percentage.
+console.log(`You got a ${gradeConvertor(yourScore)}`);
+//I wrote a function for convert the score to grade.
+function gradeConvertor(score){
```
Comment (wilgert) [inline Week2/homework/js-exercises/6_gradeConvertor.js]: Good job! One more thing, can you add some functionality to your gradeConvertor to support exams where the maximum score is different than 100?

File: Week2/homework/js-exercises/1_removeComma.js (hunk 1 above)
Comment (eeeibrahimsahin) [reply Week2/homework/js-exercises/1_removeComma.js]: you are right, my way is wrong way:). I fixed it.

File: Week2/homework/js-exercises/3_recipeCard.js (hunk 3 above)
Comment (eeeibrahimsahin) [reply Week2/homework/js-exercises/3_recipeCard.js]: I did. can you check please.

File: Week2/homework/js-exercises/6_gradeConvertor.js (hunk 6 above)
Comment (eeeibrahimsahin) [reply Week2/homework/js-exercises/6_gradeConvertor.js]: I did. can you check please.

File: Week2/homework/js-exercises/2_oddEvenReporter.js (hunk 2 above)
Comment (eeeibrahimsahin) [reply Week2/homework/js-exercises/2_oddEvenReporter.js]: yes. i fixed it.

File: Week2/homework/js-exercises/4_readBooks.js (hunk 4 above)
Comment (eeeibrahimsahin) [reply Week2/homework/js-exercises/4_readBooks.js]: thank you. i know this thing is important for clean code.

**Pull request closed without merging**